README.md

```diff
@@ -14,6 +14,19 @@ on building killer features and UI.
 - [Blog](http://emberjs.com/blog)
 - [Builds](http://emberjs.com/builds)
 
+
+The Ember.js framework enables developers to build scalable, single-page web applications gracefully and efficiently. It does so by providing a set of useful and easily incorporated functionality, the kind commonly found on dynamic, feature-rich web pages. Useful features and benefits of Ember include:
+
+* **Built-in best practices:** all code conforms to industry-standard best practices
+
+* **Rich feature set:**  Ember-specific features such as the Router and the Ember Command Line Interface enhance productivity and functionality   
+
+* **Extensive addon collection:** Ember has a large selection of available addons and plugins (over 1,500 at the time of this writing)
+
+* **[Ember Command Line Interface](https://github.com/emberjs/ember.js/)**
```

Comment:
All set, I've pushed the changes. Thanks for the feedback!

On Tue, May 17, 2016 at 8:43 PM, Ricardo Mendes <notifications@github.com>
wrote:

> In README.md
> <https://github.com/emberjs/ember.js/pull/12932#discussion_r63629231>:
>
> > @@ -14,6 +14,19 @@ on building killer features and UI.
> >  - [Blog](http://emberjs.com/blog)
> >  - [Builds](http://emberjs.com/builds)
> >
> > +
> > +The Ember.js framework enables developers to build scalable, single-page web applications gracefully and efficiently. It does so by providing a set of useful and easily incorporated functionality, the kind commonly found on dynamic, feature-rich web pages. Useful features and benefits of Ember include:
> > +
> > +* **Built-in best practices:** all code conforms to industry-standard best practices
> > +
> > +* **Rich feature set:**  Ember-specific features such as the Router and the Ember Command Line Interface enhance productivity and functionality
> > +
> > +* **Extensive addon collection:** Ember has a large selection of available addons and plugins (over 1,500 at the time of this writing)
> > +
> > +* **[Ember Command Line Interface](https://github.com/emberjs/ember.js/)**
>
> See above, also link points to the wrong repository ;)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/emberjs/ember.js/pull/12932/files/db1145284d2a735825b845c37c3feeacfd3eec8e#r63629231>
>